### PR TITLE
docs(changelog): Add 'CHANGLOG.md'

### PR DESCRIPTION
**Pull request by niheaven**
Generated by [git-chglog](https://github.com/git-chglog/git-chglog) and manually reorganized.

~Commits before https://github.com/ScoopInstaller/Scoop/releases/tag/2019-05-12 and those don't follow [Conventional Commits](https://www.conventionalcommits.org/) will be added soon.~

Hope this should be done ONLY ONCE 😭 

Ref: [Conventional Commits](https://www.conventionalcommits.org/) / [Git Commit Msg](https://karma-runner.github.io/latest/dev/making-changes.html)

## Suggested commit message:

```
<type>(<scope>): <subject>
<BLANK LINE>
<body>
<BLANK LINE>
<footer>
```

## Suggested `type`:

- **feat** (Features) for a new feature for the user, not a new feature for build script. Such commit will trigger a release bumping a MINOR version.
- **fix** (Bug Fixes) for a bug fix for the user, not a fix to a build script. Such commit will trigger a release bumping a PATCH version.
- **perf** (Performance Improvements) for performance improvements. Such commit will trigger a release bumping a PATCH version.
- **docs** (Documentation) for changes to the documentation.
- **style** (Styles) for formatting changes, missing semicolons, etc.
- **refactor** (Code Refactoring) for refactoring production code, e.g. renaming a variable.
- **test** (Tests) for adding missing tests, refactoring tests; no production code change.
- **build** (Builds) for updating build configuration, development tools or other changes irrelevant to the user.
- **ci** (Continuous Integration) for changes to CI/CD configuration files and scripts (example scopes: GitHub Actions, Excavator, etc.)
- **breaking** (BREAKING CHANGE) for breaking API changes or CLI changes
- (*not recommend*) **chores** (Chores) for changes to the build process or auxiliary tools and libraries such as documentation generation, don't modify src or test files

## Notes about `type`

- **docs**: `README.md` and some function documents
- **style**: formatting, e.g., white spaces, function case changes, output beautifying
- **refactor**: rewriting or refactoring functions
- **test**: changes in `test` directory and not in `appveyor.yml`
- **build**: changes in `bin` and `supporting` directories, plus `schema.json`
- **ci**: changes in `appveyor.yml` and (*optional*) `test` directory
- **breaking**: breaking changes that may affect manifest structure or CLI in `libexec` directory

`type` and `scope` should be chosen by what and where key change occurred.

### CODE REVIEW

**Comment by niheaven** [comment]
Ref: [Git Commit Msg](https://karma-runner.github.io/latest/dev/making-changes.html)

Suggested commit message:

```
<type>(<scope>): <subject>
<BLANK LINE>
<body>
<BLANK LINE>
<footer>
```

Suggested `type`:

- **feat** (Features) for a new feature for the user, not a new feature for build script. Such commit will trigger a release bumping a MINOR version.
- **fix** (Bug Fixes) for a bug fix for the user, not a fix to a build script. Such commit will trigger a release bumping a PATCH version.
- **perf** (Performance Improvements) for performance improvements. Such commit will trigger a release bumping a PATCH version.
- **docs** (Documentation) for changes to the documentation.
- **style** (Styles) for formatting changes, missing semicolons, etc.
- **refactor** (Code Refactoring) for refactoring production code, e.g. renaming a variable.
- **test** (Tests) for adding missing tests, refactoring tests; no production code change.
- **build** (Builds) for updating build configuration, development tools or other changes irrelevant to the user.
- **breaking** (BREAKING CHANGE) for breaking API changes or CLI changes
- (*not recommend*) **chores** (Chores) for changes to the build process or auxiliary tools and libraries such as documentation generation, don't modify src or test files

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,1315 @@
...
+- **checkurls:** Trim renaming suffix in url ([#3677](https://github.com/ScoopInstaller/Scoop/issues/3677))
+- **comspec:** Escape variables when calling COMSPEC commands ([#3538](https://github.com/ScoopInstaller/Scoop/issues/3538))
+- **decompress:** Fix bugs on extract_dir ([#3540](https://github.com/ScoopInstaller/Scoop/issues/3540))
+- **editorconfig:** Add missing } to bat/cmd regex ([#3529](https://github.com/ScoopInstaller/Scoop/issues/3529))
+- **help:** Rename help() to scoop_help() ([#3564](https://github.com/ScoopInstaller/Scoop/issues/3564))
+- **install:** Use Join-Path instead of string gluing. ([#3566](https://github.com/ScoopInstaller/Scoop/issues/3566))
+- **scoop-info:** Fix output for single binaries with alias ([#3651](https://github.com/ScoopInstaller/Scoop/issues/3651))
+- **scoop-info:** Remove a whitespace ([#3652](https://github.com/ScoopInstaller/Scoop/issues/3652))
+
+### Continuous Integrations
```
Comment (rashil2000) [inline CHANGELOG.md]:
```suggestion
### Continuous Integration
```

**Comment by niheaven** [comment]
Would full changelog be added here? Or when will I add unorganized commits log to `CHANGELOG.md` from?

IMO, commits log before 2018 should be useless now... @ScoopInstaller/maintainers

**Comment by rashil2000** [comment]
I think if you have them written already, we can add them. They're for older releases, so it doesn't matter if they're unorganised.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**